kafka.go

```diff
@@ -52,6 +52,7 @@ func (kc *kafkaClient) sendMessage(topic string, bytes []byte) (err error) {
 	return err
 }
 
+//nolint:unused
```

Comment:
why nolint these two? isn't `close` an unxported, unused method?

Reply:
It was not used before, so I added a step to actually use it with Stop method.